### PR TITLE
Fixes #1302: current version in the UI

### LINKED ISSUE #1302: Display the current MS2 version in the UI
Users have no way to tell what version of MapStore2 they are using.
Having a version displayed somewhere in the UI (even if hidden under menus, windows, tabs, or anything) is usefull when reporting a bug.

### PR DESCRIPTION
**Pull request by mbarto**

### CODE REVIEW

File: web/client/actions/version.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
```
Comment (MV88) [inline web/client/actions/version.js]: double *  and copyright

File: web/client/plugins/Version.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
```
Comment (MV88) [inline web/client/plugins/Version.jsx]: copyright 2017

File: web/client/reducers/version.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
```
Comment (MV88) [inline web/client/reducers/version.js]: remove double *

File: web/client/actions/version.js (hunk 1 above)
Comment (MV88) [inline web/client/actions/version.js]: add test

File: web/client/reducers/version.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ *
```
Comment (MV88) [inline web/client/reducers/version.js]: add test

File: .eslintrc [hunk 5]
```diff
@@ -192,7 +192,7 @@
     "react/jsx-uses-vars": 2,        // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-uses-vars.md
     "react/no-did-mount-set-state": [2, "allow-in-func"], // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-did-mount-set-state.md
     "react/no-did-update-set-state": 2, // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-did-update-set-state.md
-    "react/no-multi-comp": 2,        // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-multi-comp.md
+    "react/no-multi-comp": 0,        // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-multi-comp.md
```
Comment (Gnafu) [inline .eslintrc]: I'll use "1", as the warning will not stop the build and we can improve this later

Comment (mbarto) [inline .eslintrc]: I think we don't need this warning, this is why I removed it. Opinions?

Comment (Gnafu) [inline .eslintrc]: I agree, we can ignore this warning

**Pull request merged**